content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
```

Comment:
Given your past participation in the community, this looks like a guest post to me that should have a corresponding introduction.